src/swift/Queue.swift

```diff
@@ -330,6 +330,20 @@ public extension DispatchQueue {
 		let p = Unmanaged.passRetained(v).toOpaque()
 		dispatch_queue_set_specific(self.__wrapped, k, p, _destructDispatchSpecificValue)
 	}
+
+	#if os(Android)
+	@_silgen_name("_dispatch_thread_detach_callback_ptr")
+	private static func _dispatch_thread_detach_callback_ptr() -> UnsafeMutablePointer<(@convention(c) () -> Void)?>
+
+	public static var threadDetachCallback: (@convention(c) () -> Void)? {
```

Comment:
I think only a one-way setter should be exposed. having a getter makes the interface un-necessarily ugly.